great_expectations_provider/operators/great_expectations.py

```diff
@@ -381,11 +381,18 @@ def build_snowflake_connection_config_from_hook(self) -> Dict[str, str]:
     def build_configured_sql_datasource_config_from_conn_id(
         self,
     ) -> Datasource:
+        create_temp_table = (
+            self.conn.extra_dejson.get("create_temp_table")
+            if self.conn.extra_dejson.get("create_temp_table") is not None
+            else True
```

Comment:
Could we try something like 
```suggestion
            self.conn.extra_dejson.get("create_temp_table", True)
```

Reply:
Nice, good to know it works similar to lodash. Much cleaner xD.